### PR TITLE
Feature/tulcdm 173 missing allied war posters fields

### PR DESCRIPTION
**Pull request by skng5**
**\* Do not delete remote branch until ingest issue resolved ***
- Refactored download conformation
- Added missing show fields
- Commit 033eb66663e7ad173de3458f51fbcf46da832793 adds missing fields to datastreams and models. There is an open issue as to how we will handle ingest. Do not delete this branch until that issue is resolved.

### CODE REVIEW

File: lib/cdm_utils.rb [hunk 1]
```diff
@@ -172,47 +172,57 @@ def self.conform(doc, collection_file_name, target_dir)
...
+        ["<Call_Number/>", "<Local_Call_Number/>"],
+        ["<Audio_Filename/>", "<File_Name/>"],
+        ["<Video_Filename/>", "<File_Name/>"],
+        ["<metadata>", "<metadata>\n  <manifest>\n    <contentdm_collection_id>#{collection_file_name}</contentdm_collection_id>\n    <Rails_Root>#{Rails.root}</Rails_Root>\n    <foxml_dir>#{target_dir}</foxml_dir>\n  </manifest>"]
+      ]
+
+      # perform replacement
+      replacements.each do |r|
+        doc1 = doc
+        doc = doc1.gsub(r.first, r.last)
```
Comment (bibliotechy) [inline lib/cdm_utils.rb]: I don't understand this variable swapping. Is there a reason we can't just do  `doc.gsub!(r.first, r.last)` ?

**Comment by bibliotechy** [comment]
@skng5 Other than the variable swapping, this look good to me.

File: lib/cdm_utils.rb (hunk 1 above)
Comment (skng5) [inline lib/cdm_utils.rb]: That's probably a better way to do it.

**Comment by skng5** [comment]
gsub! Done!

**Comment by skng5** [comment]
@bibliotechy gsub! Done!

**Comment by bibliotechy** [comment]
:+1:

**Pull request merged**